client/my-sites/checkout/checkout/index.jsx

```diff
@@ -414,14 +414,17 @@ export class Checkout extends React.Component {
 			// We cannot simply compare `hostname` to `selectedSiteSlug`, since the latter
 			// might contain a path in the case of Jetpack subdirectory installs.
 			if ( adminUrl && redirectTo.startsWith( `${ adminUrl }post.php?` ) ) {
+				const plan = get( query, [ 'plan' ] );
+				const action = get( query, [ 'action' ] );
```

Comment:
Probably unnecessary, but will leave that at your discretion ( @ockham ).

Reply:
Given the state of this file, it probably makes little difference, but are these `get` adding any value/safety?

I suspect we have reasonable confidence that `query` is an object, and accessing its immediate properties should be the same behavior as `get`.

Reply:
I will check parseUrl, but I think it also returns an empty object if no params. So you are double right if that's the case.